Dockerfile

```diff
@@ -1 +1,2 @@
-FROM botpress/server:latest
\ No newline at end of file
+FROM botpress/server:latest
+CMD ["./bp", "&&" ,"./duckling"]
```

Comment:
I know we do this at multiple places (including the doc if I'm not mistaken) but why exactly do we start Botpress before duckling? Is there a particular reason or do we attain the same result either way?

Reply:
For this particular case. 

`Heroku take the first process and map the port for the HTTP.`